runtime/doc/lsp.txt

```diff
@@ -173,6 +158,25 @@ name: >
   vim.lsp.protocol.TextDocumentSyncKind.Full == 1
   vim.lsp.protocol.TextDocumentSyncKind[1] == "Full"
 
+================================================================================
```

Comment:
I am sorry to nitpick, but why rename *lsp-highlight*  to *lsp-highlight-groups* ? I guess it's fine as this hasn't been released yet but I would prefer to minimize changes as the naming may have been discussed in a previous pr, I see no reason to change/move it.

Reply:
No problem :) I realized it was duplicated so took the one with more context and details. But will rename that one to remove "groups"